packages/devtools_app/lib/src/screens/app_size/app_size_controller.dart

```diff
@@ -230,7 +234,8 @@ class AppSizeController {
     changeAnalysisJsonFile(jsonFile);
 
     // Set name for root node.
-    processedJson['n'] = 'Root';
+    if (processedJson['n'] == 'Dummy Root') isDeferredApp = true;
+    // processedJson['n'] = 'Root';
```

Comment:
I'm guessing we still want to rename this to 'Root' if it's not a deferred app

Reply:
Also we want to set `isDeferredApp` to false if we have no "dummy root". Let's also rename to `ArtificialRoot` instead of `Dummy`.